packages/overlay/test/overlay-timer.test.ts

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { useFakeTimers, SinonFakeTimers } from 'sinon';
+import { OverlayTimer } from '../lib/overlay-timer';
+import { expect } from '@open-wc/testing';
+
+describe('Overlays timer', () => {
+    let clock: SinonFakeTimers;
```

Comment:
This `SinonFakeTimers` is fancy! Lots of stuff to learn from in here.

Reply:
Yah, I was happy with how this landed. I didn't know this stuff before...